### PR TITLE
remove form from main component. add form to slideone

### PR DESCRIPTION
**Pull request by John-Paul-Larkin**
## Pull Request details

- Trying my best (against my natural instincts) to keep the PR small and manageable.

- This PR modifies the main component by removing the form. I also add a form to slideOne.
- I have two options for submitting the form, you can see in the video. I think the commented out version is better.
- Once I have this PR through i will do a few more PRs
- 1. Update the other slides in the same manner 
- 2. create the server actions 
- 3. Add logic for routing based on the null field status 

## Associated Screenshots

https://youtu.be/2BrD5Ogfrms

## Summary by CodeRabbit

- **Refactor**
	- Renamed and restructured types and schemas for handling user details, improving form handling and rendering within the app.

### CODE REVIEW

File: app/(app)/alpha/additional-details/_client.tsx [hunk 1]
```diff
@@ -39,169 +39,166 @@ export default function AdditionalSignUpDetails({
...
+      // TODO add server action logic
+      // await handleFormSlideOneSubmit(data);
+      toast.success("Saved");
       router.push(`?slide=${2}`, { scroll: false });
+    } catch (error) {
+      toast.error("An unexpected error occurred.");
     }
   };
 
+  // const onFormSubmit = async (data: TypeSlideOneSchema) => {
```
Comment (NiallJoeMaher) [inline app/(app)/alpha/additional-details/_client.tsx]: Is this comment needed?

Comment (John-Paul-Larkin) [reply app/(app)/alpha/additional-details/_client.tsx]: > Is this comment needed?

I have two options for the onFormSubmit function. Line 73 vs line 84. 
>>const isSuccess = await handleFormSlideOneSubmit(data)
Returns a boolean from the action.

My preference is for the commented out version, but I wanted your opinion on it.

Comment (NiallJoeMaher) [reply app/(app)/alpha/additional-details/_client.tsx]: Perfect! I'll look at it a little later. In meetings for the afternoon.

Comment (NiallJoeMaher) [reply app/(app)/alpha/additional-details/_client.tsx]: Go with your preference! I think that's good for the moment. 🦾

**Review by NiallJoeMaher — APPROVED** [review]
LGTM 🌮

**Pull request merged**